README.md

```diff
@@ -1,78 +1,65 @@
-| `id`       | Plugin ID. Needs to match the plugin ID in `plugin.json` |
-| `type`     | Plugin type, e.g. `panel`, `datasource`, or `app`        |
-| `url`      | URL to the plugin's GitHub project page                  |
-| `versions` | List of all published versions of the plugin             |
+## Maintain your plugin
 
-### Plugin Version Schema
+To submit an update for an already published plugin:
 
-| Property   | Description                                                 |
-|------------|-------------------------------------------------------------|
-| `version`  | Plugin version. Needs to match the version in `plugin.json` |
-| `url`      | URL to the plugin's GitHub project page                     |
-| `download` | Download information.                                       |
+1. [Sign in](https://grafana.com/auth/sign-in) to your Grafana Cloud account.
+1. In the left menu, click **My Plugins** under **Org settings**.
+1. Click **Submit Update** for the plugin you want to update.
+1. Enter the information requested by the form.
+	- **OS & Architecture:** 
+		- Select **Single** if your plugin archive contains binaries for multiple architectures.
+		- Select **Multiple** if you'd like to submit separate plugin archives for each architecture. This can lead to faster downloads since users can select the specific architecture they want to install the plugin on.
+	- **URL:** A URL that points to a ZIP archive of your packaged plugin.
+	- **MD5:** The MD5 hash of the plugin specified by the **URL**.
+1. Click **Submit**.
 
-### Plugin Download Schema
+## Guidelines
 
-| Property   | Description                                                      |
-|------------|------------------------------------------------------------------|
-| `url`      | URL to a ZIP archive containing a production build of the plugin |
-| `md5`      | MD5 check sum of the ZIP archive                                 |
+To speed up the time it takes to review your plugin:
 
-## Resources
+- Check that your plugin is ready for review using the [plugin validator](https://plugin-validator.grafana.net).
+- Read our [6 tips for improving your Grafana plugin before you publish](https://grafana.com/blog/2021/01/21/6-tips-for-improving-your-grafana-plugin-before-you-publish/)
 
-- [Plugin Development](https://grafana.com/docs/grafana/latest/developers/plugins/)
-- [Sign a plugin](https://grafana.com/docs/grafana/latest/developers/plugins/sign-a-plugin)
-- [Package a plugin](https://grafana.com/docs/grafana/latest/developers/plugins/package-a-plugin)
-- [plugin.json Schema](https://grafana.com/docs/grafana/latest/developers/plugins/metadata/)
-- [6 tips for improving your Grafana plugin before you publish](https://grafana.com/blog/2021/01/21/6-tips-for-improving-your-grafana-plugin-before-you-publish/)
+## FAQ
+
+- **Do I need to submit a private plugin?:** No. Please only submit plugins that you wish to make publicly available for the Grafana community.
```

Comment:
Maybe change to **Can I submit a private plugin?:** since it seems like we don't want folks to.